src/js/i18n/definitions.js

```diff
@@ -578,3 +596,18 @@ export const news = defineMessages({
     defaultMessage: 'Published {date}',
   },
 });
+
+export const filter = defineMessages({
+  filter: 'Filter',
+  filters: 'Filters',
+  date: 'Date',
+  from: 'From',
+  to: 'To',
+  month: 'Month',
+  day: 'Day',
+  year: 'Year',
+  clearFilters: 'Clear filters',
+  applyFilters: 'Apply filters',
+  filteredResults: '{count} filtered results',
+  OneFilteredResult: '1 filtered result',
```

Comment:
the nitpickiest thing I'll say is `oneFilteredResult`

Reply:
you're right!